### ISSUE TITLE
Certificate error during fetch of .deb package

### ISSUE THREAD

**Issue by gibbsoft** [lifeofguenter/ansible-role-php5-fpm#1]
I'm getting this error on `Ubuntu 14.0.4`, which I originally put down to the version of python (`2.7.6`).  However I have now tried this with `2.7.10` and `2.7.13`, even on `Debian 8` and I get the same result:

```
TASK [lifeofguenter.php5-fpm : Fetch .deb] ********************************************************************************************
task path: /opt/cm/ansible/roles/theirs/lifeofguenter.php5-fpm/tasks/php-fpm.yml:29
Friday 01 September 2017  09:21:54 +0000 (0:00:00.462)       0:01:00.602 ******
 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: php5fpm_current.rc != 0 or
php5fpm_current.stdout.find("PHP {{ php5fpm_real_version }} (fpm-fcgi)") == -1
Using module file /root/.pyenv/versions/2.7.13/lib/python2.7/site-packages/ansible/modules/network/basics/get_url.py
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c '/usr/bin/python && sleep 0'
fatal: [localhost]: FAILED! => {
    "changed": false,
    "failed": true,
    "invocation": {
        "module_args": {
            "attributes": null,
            "backup": false,
            "checksum": "",
            "content": null,
            "delimiter": null,
            "dest": "/tmp/php5-fpm_5.6.31-31_amd64.deb",
            "directory_mode": null,
            "follow": false,
            "force": false,
            "force_basic_auth": false,
            "group": null,
            "headers": null,
            "http_agent": "ansible-httpget",
            "mode": null,
            "owner": null,
            "regexp": null,
            "remote_src": null,
            "selevel": null,
            "serole": null,
            "setype": null,
            "seuser": null,
            "sha256sum": "",
            "src": null,
            "timeout": 10,
            "tmp_dest": "",
            "unsafe_writes": null,
            "url": "https://ftp.weheartwebsites.de/ubuntu/trusty/php5-fpm_5.6.31-31_amd64.deb",
            "url_password": null,
            "url_username": null,
            "use_proxy": true,
            "validate_certs": true
        }
    },
    "msg": "Failed to validate the SSL certificate for ftp.weheartwebsites.de:443. Make sure your managed systems have a valid CA certificate installed. If the website serving the url uses SNI you need python >= 2.7.9 on your managed machine or you can install the `urllib3`, `pyOpenSSL`, `ndg-httpsclient`, and `pyasn1` python modules to perform SNI verification in python >= 2.6. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible. The exception msg was: hostname 'ftp.weheartwebsites.de' doesn't match either of 'www.annuairetel247.com', 'annuairetel247.com'."
}
```

Is anybody else experiencing this issue?

**Comment by gibbsoft** [comment]
I've submitted PR#2 for this as a temporary solution for anyone else facing broken builds.  `ansible-role-php5-cli` has the same issue.

**Comment by lifeofguenter** [comment]
fixed via #2
